### PR TITLE
remove docs page

### PR DESCRIPTION
**Pull request by volodymyrmakarchuk-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-363923](https://oktainc.atlassian.net/browse/OKTA-363923)

### CODE REVIEW

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
When attempting to test this change locally, a redirect did not occur and instead 304s were returned. I tried in Chrome private mode and also in safari which I never used. It seems the 304 is a server cache response. See images below:

**Page I saw**
image


**304 Server Response:**
image

I think maybe the server cache needs to be flushed of the page. Would this happen in production and is there are any other changes that need to be made?

**Comment by volodymyrmakarchuk-okta** [comment]
@robertnicolo-okta Redirection is not working locally, please check trex configuration instead
https://vuepress-site.trexcloud.com/

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
Hi,

I retested using the "https://vuepress-site.trexcloud.com/" site and noticed two redirect bounces before ending up at the homepage.  See screenshot:

image

**Comment by volodymyrmakarchuk-okta** [comment]
@robertnicolo-okta seems this behavior is a common pattern for SEO optimization, 301 redirections on trailing slash version prevent content duplication, here I found reasonable article https://www.onlineassetpartners.co.nz/blog/how-trailing-slashes-affect-your-websites-seo/

Also, you may want to check conductor.yml file. This file contains settings for route redirects, and we use one approach for all redirects so redirect behavior with 301 code seems applicable for all routes

File: packages/@okta/vuepress-site/docs/index.md [hunk 1]
```diff
@@ -1,111 +0,0 @@
----
-# This file controls the links and content shown on the /docs landing page.
-
-component: Documentation
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-site/docs/index.md:4]: See if the vue component can be deleted, too. It's probably the only page that uses it

**Review by shawyu-okta — APPROVED** [review]

**Review by shawyu-okta — APPROVED** [review]

**Review by robertnicolo-okta — APPROVED** [review]
lgtm

**Pull request merged**